code/_helpers/lists.dm

```diff
@@ -216,6 +216,12 @@ Checks if a list has the same entries and values as an element of big.
 	for(var/e in L)
 		. += L[e]
 
+// Return a list of the keys in an assoc list
+/proc/list_keys(var/list/L)
```

Comment:
As mentioned on Discord I think this should probably have a more specific name, as `list_keys` could be misleading about what it actually does.

Reply:
Removed `list_keys()`.